examples/windows/windows-perfmon-proc-owner.yml

```diff
@@ -0,0 +1,13 @@
+integrations:
+  - name: nri-flex # We're telling the Infra agent to run Flex!
+    interval: 60s
+    timeout: 45s
+    config: # Flex configuration starts here!
+      name: msPerfProcOwner
+      apis:
+        # Event Requests
+        - event_type: msPerfProcOwner # Name of the event in New Relic
+          commands:
+            - run: "& \"C:/Program Files/New Relic/newrelic-infra/integrations.d/windows-perfmon-proc-owner\""
```

Comment:
missing the file extension on this command

Reply:
works fine like this ... referred to this https://github.com/newrelic/nri-flex/blob/master/examples/windows/windows-ping-example.yml